### PR TITLE
Lf wf 86828952 standardize repo

### PR DESCRIPTION
**Pull request by lfender6445**
standardize package with yo generator, bump version

will be used with tealium

### CODE REVIEW

File: dist/browser-detect.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
+define(['browser-detect'], function() {
```
Comment (pasha) [inline dist/browser-detect.js]: @lfender6445 what are you thoughts on adding few more helper methods, AG just had a case that we had to put a wrapper around one of the calls `.platform` https://github.com/rentpath/ag/commit/9ccb8c8380df30ccc0fed90d38fe457a6050c426
in ag-utils.js

Comment (lfender6445) [inline dist/browser-detect.js]: good idea

**Comment by c0** [comment]
Just a heads up — tagging in a PR can make it difficult to track down changes. m.AG broke b/c it's version of jquery-autotagging (an old version) uses the latest version of this repo (which has a `dist/` dir).

File: app/coffeescript/browser-detect.coffee [hunk 2]
```diff
@@ -30,6 +30,12 @@ define 'browserdetect', ->
       return if index == -1
       return parseFloat(dataString.substring(index + versionLabel.length + 1))
 
+    @isExplorer: ->
+      return BrowserDetect.platform().browser == 'explorer'
+
+    @isExplorer8: ->
+      return BrowserDetect.isExplorer() && BrowserDetect.platform().version == '8.0'
```
Comment (lfender6445) [inline app/coffeescript/browser-detect.coffee]: i think we can replace all instances of `BrowserDetect` with `@` which would be more coffeesque

**Comment by pasha** [comment]
:+1: @c0 are you talking about versioning of the package?, if yes i agree, i prefer a separate commit for just version update.

**Comment by lfender6445** [comment]
i just spoke to caleb - we got greedy earlier by pushing tag before PR was merged, thats what caleb was referring to.

**Comment by pasha** [comment]
that's sounds good @lfender6445

**Comment by lfender6445** [comment]
:+1:

**Pull request merged**